marda_extractors_api/__init__.py

```diff
@@ -234,14 +237,14 @@ def execute(
         return output
 
     def _execute_cli(self, command):
-        print(f"Exexcuting {command=}")
+        print(f"Executing {command=}")
         results = subprocess.check_output(command, shell=True)
         return results
 
     def _execute_cli_venv(self, command):
-        print(f"Exexcuting {command=} in venv")
-        py_cmd = "import subprocess; return subprocess.check_output(f'{command}', shell=True)"
-        command = [str(self.venv_dir / "bin" / "python"), "-c", py_cmd]
+        print(f"Executing {command=} in venv")
+        py_cmd = f"import subprocess; subprocess.check_output(r'{command}', shell=True)"
```

Comment:
```suggestion
        py_cmd = f"import subprocess; subprocess.check_output(r'{command}', shell=True)"
```

Haha thanks, I wrote this as my Eurostar started boarding and then wasted the entire journey just re-running the tests without making any changes and scratching my head...